src/v2/view-builder/utils/AutoRedirectUtil.js

```diff
@@ -0,0 +1,18 @@
+import Util from '../../../util/Util';
+import {BaseHeader} from '../internals';
+import {BaseAuthenticatorBeacon} from '../components/BaseAuthenticatorView';
+import {AUTHENTICATOR_KEY} from '../../ion/RemediationConstants';
+
+function getHeader() {
```

Comment:
We show the following in the view as requested by @stevennguyen-okta 
![Screenshot 2023-09-28 at 1 25 13 AM](https://github.com/okta/okta-signin-widget/assets/83233975/73309bb7-9a92-4091-8579-3535c250ad86)